source/agora/script/Engine.d

```diff
@@ -735,6 +755,144 @@ public class Engine
         sig_valid = Schnorr.verify(point, sig, tx);
         return null;
     }
+
+    /***************************************************************************
+
+        Checks floating-transaction signatures for use with the Flash layer.
+
+        Verifies the sequence signature by blanking the input, reading the
+        minimum sequence, the key, the new sequence, and the signature off
+        the stack and validates the signature.
+
+        If any of the arguments expected on the stack are missing,
+        an error string is returned.
+
+        The `sig_valid` parameter will be set to the validation result
+        of the signature.
+
+        Params:
+            OP = the opcode
+            stack = should contain the Signature and Public Key
+            tx = the transaction that should have been signed
+            input = the associated Input to blank when signing
+            sig_valid = will contain the signature validation result
+
+        Returns:
+            an error string if the needed arguments on the stack are missing,
+            otherwise returns null
+
+    ***************************************************************************/
+
+    private string verifySequenceSignature (OP op)(ref Stack stack,
+        in Transaction tx, in Input input, out bool sig_valid)
+        nothrow @safe //@nogc  // stack.pop() is not @nogc
```

Comment:
Do you intend to leave the comment?

Reply:
Yes because I wanted to make `pop()` `@nogc` in the future.

But maybe these comments are not so useful?

Reply:
Yes, useful.